.reuse/dep5

```diff
@@ -0,0 +1,12 @@
+Format: https://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
+Upstream-Name: ohwr.org
+Upstream-Contact: CERN (home.cern)
+Source: https://github.com/OHWR/ohwr.org
+
+Files: static/images/*
+Copyright: 2023 CERN (home.cern)
+License: CC-BY-SA-4.0+
+
+Files: go.sum
+Copyright: 2023 CERN (home.cern)
+License: BSD-3-Clause
```

Comment:
I think you can remove this file from the repo and gitignore this file. I did it as a test and running `hugo` regenerates it.

Reply:
yes, go.sum is generated automatically from go.mod but it's one of those rare cases where it's a good practice to commit an automatically generated file.

https://github.com/golang/go/wiki/Modules#should-i-commit-my-gosum-file-as-well-as-my-gomod-file

Reply:
Awesome, thanks for the explanation.